File: WWWRubberCityFoundation/src/app/models/local-help.ts
export class LocalHelp {
    id: number;
    linkUrl: string;
    description: string;
    email: string;
    phone: string;

    constructor(id: number, linkUrl: string, description: string, email: string, phone: string) {
        this.id = id;
        this.linkUrl = linkUrl;
        this.description = description;
        this.email = email;
        this.phone = phone;
    }
}

File: Domain/Models/LocalHelp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class LocalHelp
    {
        public int Id { get; set; }
        public string linkUrl { get; set; }
        public string description { get; set; }
        public string email { get; set; }
        public string phone { get; set; }
    }
}
